tests/build.go

```diff
@@ -187,6 +190,42 @@ RUN busybox | head -1 | grep v1.35.0
 	require.NoError(t, cmd.Run())
 }
 
+func testBuildDetailsLink(t *testing.T, sb integration.Sandbox) {
+	buildDetailsPattern := regexp.MustCompile(`(?m)^View build details: docker-desktop://dashboard/build/[^/]+/[^/]+/[^/]+\n$`)
+
+	// create desktop-build .lastaccess file
+	home, err := os.UserHomeDir() // TODO: sandbox should create a temp home dir and expose it through its interface
+	require.NoError(t, err)
+	dbDir := path.Join(home, ".docker", "desktop-build")
+	require.NoError(t, os.MkdirAll(dbDir, 0755))
+	dblaFile, err := os.Create(path.Join(dbDir, ".lastaccess"))
```

Comment:
Could we run the simple/erroneous builds before this as well, and check that we *don't* get the build details? Then we have a test to make sure this doesn't leak into non desktop-build scenarios.

Reply:
Good point will add this